File: Python/pr_3.py
"""
   1
  2 3
 4 5 6
7 8 9 10
"""

def space(n, i):
    for j in range(1, n + 1 - i):
        print(" ", end="")

def main():
    n, c = 4, 1
    for i in range(1, n + 1):
        space(n, i)
        for j in range(1, i + 1):
            print(c, end=" ")
            c += 1
        print()

if __name__ == "__main__":
    main()

File: C/pr_3.c
#include <stdio.h>

void space(int n, int i) {
    for (int j = 1; j <= n - i; j++) {
        printf(" ");
    }
}

int main() {
    int n = 4;
    int c = 1;
    for (int i = 1; i <= n; i++) {
        space(n, i);
        for (int j = 1; j <= i; j++) {
            printf("%d ", c);
            c++;
        }
        printf("\n");
    }

    return 0;
}
